tests/processor/test_tasks.py

```diff
@@ -146,16 +157,125 @@ def test_get_report_files_second_task_not_called(self,
+    @patch('masu.processor.tasks._get_report_files')
+    @patch('masu.processor.tasks.process_report_file')
+    def test_get_report_files_timestamps_empty(self,
+                                               mock_process_files,
+                                               mock_get_files,
+                                               mock_started,
+                                               mock_completed):
+        """
+        Test that the chained task is called when no start time is set.
+        """
+        mock_process_files.delay = Mock()
+        mock_get_files.return_value = self.fake_reports
+
+        mock_started.return_value = None
+        mock_completed.return_value = self.today
+        get_report_files(**self.fake_get_report_args)
+        mock_process_files.delay.assert_called()
+
+    @patch('masu.processor.tasks.ReportStatsDBAccessor.get_last_completed_datetime')
+    @patch('masu.processor.tasks.ReportStatsDBAccessor.get_last_started_datetime')
+    @patch('masu.processor.tasks._get_report_files')
+    @patch('masu.processor.tasks.process_report_file')
+    def test_get_report_files_timestamps_empty(self,
+                                               mock_process_files,
+                                               mock_get_files,
+                                               mock_started,
+                                               mock_completed):
+        """
+        Test that the chained task is called when no end time is set.
+        """
+        mock_started.return_value = self.today
+        mock_completed.return_value = None
+        get_report_files(**self.fake_get_report_args)
+        mock_process_files.delay.assert_called()
+
+    @patch('masu.processor.tasks.ReportStatsDBAccessor.get_last_completed_datetime')
+    @patch('masu.processor.tasks.ReportStatsDBAccessor.get_last_started_datetime')
+    @patch('masu.processor.tasks._get_report_files')
+    @patch('masu.processor.tasks.process_report_file')
+    def test_get_report_files_timestamps_empty(self,
+                                               mock_process_files,
+                                               mock_get_files,
+                                               mock_started,
+                                               mock_completed):
+        """
+        Test that the chained task is called when no timestamps are set.
+        """
+        mock_started.return_value = None
+        mock_completed.return_value = None
+        get_report_files(**self.fake_get_report_args)
+        mock_process_files.delay.assert_called()
+
+    @patch('masu.processor.tasks.ReportStatsDBAccessor.get_last_completed_datetime')
+    @patch('masu.processor.tasks.ReportStatsDBAccessor.get_last_started_datetime')
+    @patch('masu.processor.tasks._get_report_files')
+    @patch('masu.processor.tasks.process_report_file')
+    def test_get_report_files_timestamps_empty(self,
```

Comment:
This one seems to be a duplicate of the first `test_get_report_files_timestamps_empty` above.